### ISSUE TITLE
4.07 support

### ISSUE THREAD

**Issue by copy** [janestreet/base#47]
Currently, on 4.07.0+beta2+flambda:

```
jbuilder build @install
      ocamlc src/.base.objs/base__Formatter.{cmi,cmti} (exit 2)
(cd _build/default && /media/samsung/opam2/4.07.0+beta2+flambda/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.base.objs -I /media/samsung/opam2/4.07.0+beta2+flambda/lib/sexplib0 -I compiler-stdlib/src/.caml.objs -I shadow-stdlib/src/.shadow_stdlib.objs -no-alias-deps -open Base__ -o src/.base.objs/base__Formatter.cmi -c -intf src/formatter.mli)
File "src/formatter.mli", line 9, characters 9-30:
Error: Unbound module Caml.Format
      ocamlc src/.base.objs/base__Import0.{cmi,cmo,cmt} (exit 2)
(cd _build/default && /media/samsung/opam2/4.07.0+beta2+flambda/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.base.objs -I /media/samsung/opam2/4.07.0+beta2+flambda/lib/sexplib0 -I compiler-stdlib/src/.caml.objs -I shadow-stdlib/src/.shadow_stdlib.objs -no-alias-deps -open Base__ -o src/.base.objs/base__Import0.cmo -c -impl src/import0.ml)
File "src/import0.ml", line 7, characters 6-394:
Error: The signature constrained by `with' has no component named Pervasives
```

**Comment by diml** [comment]
Indeed, it needs a patch since the introduction of `Stdlib`. I believe @hhugo has a patch for this

**Comment by kit-ty-kate** [comment]
Any news on that now that 4.07 has been released ?

**Comment by hhugo** [comment]
Will look into that today

**Comment by diml** [comment]
This is fixed in the v0.11 branch. A similar fix will be pushed to master soon. We did a minor release of base: https://github.com/ocaml/opam-repository/pull/12339

**Comment by copy** [comment]
Thanks, can confirm that it's working. Follow-up: https://github.com/janestreet/core_kernel/issues/84.
